### PR TITLE
DSEthiop 1 to 6

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
nearly finished? Was sagst du?

### CODE REVIEW

**Comment by DenisNosnitsin1970** [comment]
OK, not exactly the end, because there is still the collation issue :) + the term "doublure" for which I will find another term tomorrow :)

File: DayrSuryan/DSEthiop1.xml [hunk 1]
```diff
@@ -672,15 +672,10 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                             <locus target="#7"/>, <locus target="#13"/>, <locus target="#20"/>, <locus target="#29"/>,
-                             <locus target="#35"/>,
-                             <locus target="#48"/>, <locus target="#57"/>, <locus target="#63"/>, <locus target="#68"/>,
-                             <locus target="#78"/>, <locus target="#91"/>, <locus target="#96"/> , <locus target="#102"/>, <locus target="#112"/> (red),
-
-                             <locus target="#127"/>.
-                             A loose red thread is placed between <locus target="#20"/> and <locus target="#21"/>.
+                         Leaf string markers (coloured threads) inserted in the upper part of
+                         <locus target="#1 #7 #13 #20 #29 #35 #48 #57 #63 #68 #78 #91 #96 #102"/>,
+                         <locus target="#112"/> (red), and <locus target="#127"/>.
```
Comment (thea-m) [inline DayrSuryan/DSEthiop1.xml:677]: I think it could be "...#102 #112 #127". Probably the information that one is red does not need to be kept?

Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop1.xml:677]: Pity, but I will remove it, otherwise it is uneven. One can figure out that it is red.

Comment (thea-m) [reply DayrSuryan/DSEthiop1.xml]: Aha, if you like the colours they can all be restituted, it was orinigally like this:
```
 Leaf string markers (coloured threads) inserted in the upper part of <locus target="#1"/>  (yellow),
                             <locus target="#7"/> (red), <locus target="#13"/> (red), <locus target="#20"/> (yellow), <locus target="#29"/> (red), <locus target="#35"/> (yellow),
                             <locus target="#48"/> (yellow), <locus target="#57"/> (pale green), <locus target="#63"/> (pale green), <locus target="#68"/> (pale green),
                             <locus target="#78"/> (yellow), <locus target="#91"/> (red), <locus target="#96"/>  (red), <locus target="#102"/> (red), <locus target="#112"/> (red),
                             <locus target="#127"/> (pale green).
```
I deleted them for readability.

Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop1.xml]: Aha, ok, this is a different matter and changes the picture a little bit. If we have this, we can keep separate fols. I will restitute them, it is interesting.

Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop1.xml]: Like this? A kind of compromise.

**Pull request merged**